templates/zerver/integrations/slack.md

```diff
@@ -0,0 +1,73 @@
+Mirror a Slack channel in Zulip!
+
+If you only want to bridge one channel in Slack with one topic in Zulip, you
+can use [Matterbridge](https://github.com/42wim/matterbridge).
```

Comment:
Is using Matterbridge the recommended approach in this simple case? If so, we should make that even more explicit.

Reply:
I rephrased the sentence.